modules/guides/examples/javascript-udfs/create-reference-function.sh

```diff
@@ -0,0 +1,3 @@
+curl -v http://localhost:8093/query/service \
+  -u Administrator:password \
+  -d 'statement=CREATE FUNCTION GetBusinessDays(startDate, endDate) LANGUAGE JAVASCRIPT AS "getBusinessDays" AT "my-library"'
```

Comment:
Best practice: finish all N1QL statements with a semicolon

```suggestion
  -d 'statement=CREATE FUNCTION GetBusinessDays(startDate, endDate) LANGUAGE JAVASCRIPT AS "getBusinessDays" AT "my-library";'
```